scripts/mkcloud

```diff
@@ -15,6 +15,19 @@
 # 3. Please 'export' environment variables according to your needs.
 # 4. read the usage: mkcloud help
 
+# include separate bash libs
+function include_bash_libs()
+{
+    local path=$(dirname $0)
+    local scripts="libvirt.sh"
```

Comment:
This is an invitation to only move code outside. Our goal is to create separate independent scripts that can be reused not only in mkcloud context.

No need to change the code here. But maybe a prominent comment might be good to point out that this mechanism is only a temporary solution during refactoring.